data/json/recipes/food/dairy_products.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "type": "recipe",
+    "result": "raw_butter",
+    "result_mult": 1,
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DAIRY",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": "30 m",
+    "qualities": [ { "id": "CHURN", "level": 1 } ],
+    "autolearn": true,
+    "components": [ [ [ "milk_cream", 5 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "milk_cream_rising",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DAIRY",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": "5 m",
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "components": [ [ [ "jar_3l_glass", 1 ] ], [ [ "milk_raw", 12 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "milk_cream",
+    "result_mult": 5,
+    "byproducts": [ [ "buttermilk", 7 ] ],
```

Comment:
Currently buttermilk is only 1 calorie. Could you please fix that in this PR?

[USDA](https://fdc.nal.usda.gov/fdc-app.html#/food-details/172225/nutrients) gives 152 kcal per cup (or ~161 calories per 250mL) for whole, fluid buttermilk.

Reply:
And done.  Thanks!